tests/e2e-udx/udx-java/51-verify-java-uda.yaml

```
@@ -0,0 +1,56 @@
+# (c) Copyright [2021-2022] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: script-verify-java-uda
+data:
+  entrypoint.sh: |-
+    #!/bin/bash
+    set -o xtrace
+    set -o errexit
+
+    VDB_NAME=v-udx-java
+    SELECTOR=app.kubernetes.io/name=vertica,app.kubernetes.io/instance=$VDB_NAME
+    POD_NAME=$(kubectl get pods --selector=$SELECTOR -o jsonpath="{.items[0].metadata.name}")
+    echo $POD_NAME
```

I suggest hard coding the pod name.
```suggestion
    POD_NAME=v-udx-java-sc1-0
```